packages/editor/src/components/multi-selection-inspector/index.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * WordPress dependencies
+ */
+import { sprintf, __ } from '@wordpress/i18n';
+import { withSelect } from '@wordpress/data';
+import { serialize } from '@wordpress/blocks';
+import { count as wordCount } from '@wordpress/wordcount';
+import {
+	Path,
+	SVG,
+} from '@wordpress/components';
+
+/**
+ * Internal Dependencies
+ */
+import BlockIcon from '../block-icon';
+
+function MultiSelectionInspector( { blocks } ) {
+	return (
+		<div className="editor-block-inspector__card">
```

Comment:
Instead of reusing a className, I'd just duplicate this style in a `style.scss` file specific to this component and rename this class `editor-multi-selection-inspector__card`.

Later, we should think at a generic card component or something.

Reply:
The other classNames should also be renamed.